src/main/resources/assets/minecolonies/lang/en_US.lang

```diff
@@ -179,3 +179,8 @@ com.minecolonies.gui.workerHuts.farmerHut.unused=unused
 com.minecolonies.gui.scarecrow.user=Field used by: %s
 com.minecolonies.gui.scarecrow.user.noone=no-one
 com.minecolonies.gui.buildtool.nohutininventory=The hut was not found in your inventory.
+
+com.minecolonies.gui.debugScreen.noCloseColony=No close colony!
+com.minecolonies.gui.debugScreen.nextColony=Next colony is %d blocks away.
```

Comment:
Could be "Next colony is %d blocks away (%d required to place a colony)", I don't think the periods are necessary.